### PR TITLE
#524: Removed custom sorting with SizeMap

### PR DESCRIPTION
**Pull request by bonkey**
#524: Removed custom sorting with SizeMap

### CODE REVIEW

File: AtlasDemo/AtlasDemo/DemoArticle.swift [hunk 1]
```diff
@@ -254,8 +249,5 @@ public func == (lhs: DemoArticle.Unit, rhs: DemoArticle.Unit) -> Bool {
 }
 
 public func < (lhs: DemoArticle.Unit, rhs: DemoArticle.Unit) -> Bool {
```
Comment (hani-ibrahim) [inline AtlasDemo/AtlasDemo/DemoArticle.swift]: does this method actually needed?

**Comment by hani-ibrahim** [comment]
👍

File: AtlasDemo/AtlasDemo/DemoArticle.swift (hunk 1 above)
Comment (bonkey) [inline AtlasDemo/AtlasDemo/DemoArticle.swift]: When we remove sorting – no. So I removed.

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**